### ISSUE TITLE
Per user boards

### ISSUE THREAD

**Issue by gusarg81** [kiswa/TaskBoard#537]
Would be nice to have per users boards instead just one admin user allow to create "global" boards.

In other words, user independent boards.

**Comment by kiswa** [comment]
TaskBoard was designed with the idea that there is one global admin, as many board admins as you want, and standard users.

If you want a board to be for a single user, only add that user to the board, and don't use the admin account for anything other than administrative tasks.

If I've misunderstood what you're looking for, please comment.

**Comment by gusarg81** [comment]
Yes. For example in this case I am the admin and i have to create boards for other users instead of that user create their own boards.

**Comment by kiswa** [comment]
Ah, so you want the ability to have a user that is allowed to create boards, and is then limited to the boards they've created?

Interesting use-case, I'll think about this for a bit.

**Comment by gusarg81** [comment]
Yes, exactly.
